docs/en/docs/kant/user-guide/device/index.md

```diff
@@ -4,7 +4,11 @@
 
 End device can range from sensors and controllers to smart cameras or industrial computers.
 
-End device can be connected to edge nodes, supporting access through the Modbus protocol and unified management.
+End device can be connected to edge nodes. You can connect them through the Modbus protocol and manage them centrally.
+
+## Device Model
+
+A device model is a digital representation of an end device in the cloud. It abstracts the common characteristics of the device into a data model, using properties to define the device and the information it can provide externally.
```

Comment:
```suggestion
A device model is a digital representation of an end device in the cloud. It abstracts the common characteristics
of the device into a data model, using properties to define the device and the information it can provide externally.
```